alkira/resource_alkira_connector_ipsec.go

```diff
@@ -281,10 +282,10 @@ func resourceAlkiraConnectorIPSec() *schema.Resource {
 				Required:    true,
 			},
 			"size": &schema.Schema{
-				Description:  "The size of the connector. one of `SMALL`, `MEDIUM` and `LARGE`.",
+				Description:  "The size of the connector. one of `SMALL`, `MEDIUM`, `LARGE`, `2LARGE`, `4LARGE`, `5LARGE`, `10LARGE` and `20LARGE`.",
```

Comment:
Based on previous comments the first period in this description should be a comma.

Reply:
Good catch 👍